packages/plugin/src/lib/localModlistSubscriptions.ts

```diff
@@ -0,0 +1,26 @@
+import { ModListSubscribeRequest } from "@mass-block-twitter/server";
+
+
+
+export async function setSubscription(modListId: string, action: ModListSubscribeRequest['action']) {
+  localStorage.setItem(`rxliuli:modlist:${modListId}:action`, action)
```

Comment:
Please use idb-keyval to store in indexedDB, as localStorage has very limited space.